### DISCUSSION TITLE
DatasetAdapter com TClientDataset

### DISCUSSION THREAD

**Discussion by MiltonSonoda** [viniciussanchez/RESTRequest4Delphi#88] [Q&A]
Vinícius, DatasetAdapter funciona com TClientDataset? Tem algum passo a mais? Com TFDMemTable Ok.

**Comment by viniciussanchez** [comment]
Sim, qualquer classe que herde de TDataSet. Talvez o que não vai funcionar é o mestre detalhe, isso não lembro... teria que dar uma olhada
